openmdao/core/indepvarcomp.py

```diff
@@ -277,7 +277,7 @@ def add_output(self, name, val=1.0, units=None):
             'res_units': None,
             'desc': '',
             'distributed': False,
-            'tags': set(['openmdao:allow_desvar']),
+            'tags': set(['openmdao:allow_desvar', 'indep_var']),
```

Comment:
I think we should "namespace" this tag (`openmdao:indep_var`) to make it clear that it is a tag recognized by OpenMDAO itself and not something the user should use in their own models unless they know what they're doing.

Reply:
fixed